ai21_tokenizer/tokenizer_factory.py

```diff
@@ -1,22 +1,18 @@
+import os
 from pathlib import Path
-from typing import Dict, Any
 
 from ai21_tokenizer.base_tokenizer import BaseTokenizer
+from ai21_tokenizer.jamba_instruct_tokenizer import JambaInstructTokenizer
 from ai21_tokenizer.jurassic_tokenizer import JurassicTokenizer
-from ai21_tokenizer.utils import load_json
+from ai21_tokenizer.utils import PathLike
 
 _LOCAL_RESOURCES_PATH = Path(__file__).parent / "resources"
-_MODEL_EXTENSION = ".model"
-_MODEL_CONFIG_FILENAME = "config.json"
+_MODEL_CACHE_DIR = _LOCAL_RESOURCES_PATH / "cache"
```

Comment:
Is that where HF are caching weights? in the source code?
Maybe add the option to override this with an env var

Reply:
I could use /tmp instead actually and use an env var. what do you think?